pyproject.toml

```diff
@@ -4,6 +4,7 @@
 
 [tool.setuptools]
   packages = ["cats"]
+  package-data.cats = ["config.yml"]
```

Comment:
I think `config.yml` would end up in `lib/python3.X/site-packages/cats` ? Might not be straightforward to use it from there?